run.sh

```diff
@@ -1,2 +0,0 @@
-npx tailwindcss -i ./input.css -o ./style/output.css
```

Comment:
How does this work now then? 

The instructions for leptos-tailwind show that you need this step: https://github.com/leptos-rs/leptos/tree/main/examples/tailwind

Reply:
Tailwind will run in watch mode when running `cargo leptos watch`. I was having trouble using the run script and had to run `cargo leptos watch` and `npx tailwind -i ./input.css -o ./style/output.css --watch`in two separate terminal sessions. If I just ran `run.sh`, tailwind wouldn't update if I added any classes to a component and I would have to kill the script and run it again.

This PR makes it so that running `cargo leptos run` will always pick up changes both in leptos and tailwind.

Reply:
Tailwind will also be automatically run during `cargo leptos build`. If you look at this output:
```
    Finished dev [unoptimized + debuginfo] target(s) in 0.12s
       Cargo finished cargo build --package=lemmy-ui-leptos --lib --target-dir=target/front --target=wasm32-unknown-unknown --no-default-features --features=hydrate
    Tailwind finished tailwind --input style/tailwind.css --config ./tailwind.config.js
    Finished dev [unoptimized + debuginfo] target(s) in 0.14s
       Cargo finished cargo build --package=lemmy-ui-leptos --bin=lemmy-ui-leptos --target-dir=target/server --no-default-features --features=ssr
```

You will see that tailwind generates the css.

Reply:
mmmk cool, thx.